itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/txn/compactor/TestCompactor.java

```diff
@@ -168,6 +170,8 @@ public void setup() throws Exception {
       }
     }
     createTestDataFile(BASIC_FILE_NAME, input);
+
+    statsUpdater = new StatsUpdater();
```

Comment:
why not initialize it right away in the field declaration?

Reply:
Sure.

Reply:
it's still in the before method ....

Reply:
Moved.